survey_admin/views.py

```diff
@@ -308,7 +354,7 @@ def show_surveys():
 @app.route('/showquestions', methods=['GET'])
 @login_required
 def show_questions():
-    question=Question.query.all()
+    question=Question.query.order_by('question_id').all()
```

Comment:
Confirmed with @eileenpeppard that showing the question ids is her preferred way to go. She is also aware that there might be gaps in the ids. This is fine with her.

Reply:
don't accept this yet, there is one more thing i need to fix. there is an error that's popping up only now for some reason because of an unused variable. ill open an issue and just add the commit to this PR (i have no idea why this little bug didn't occur on my computer or the server until now because it is as simple as a dict key not existing)

Reply:
@carlosparadis, okay I took care of that error I mentioned above, you can take a look at this whenever you get the chance. thanks!